orgues/models.py

```diff
@@ -249,6 +249,10 @@ class Orgue(models.Model):
     # Localisation
     code_dep_validator = RegexValidator(regex='^(97[12346]|0[1-9]|[1-8][0-9]|9[0-5]|2[AB])$',
                                         message="Renseigner un code de département valide")
+    latitude_validator = RegexValidator(regex='^([-]*([0-9]|[0-8][0-9])[.][0-9]*)$',
+                                        message="Renseigner une latitude valide")
+    longitude_validator = RegexValidator(regex='^([-]*([0-9]|[0-8][0-9]|1[0-7][0-9])[.][0-9]*)$',
```

Comment:
```suggestion
    longitude_validator = RegexValidator(regex='^([-]?([0-9]|[0-8][0-9]|1[0-7][0-9])[.][0-9]*)$',
```
Et pourquoi on ne peut pas avoir des nombres entre 90 et 100 ?

Reply:
Un copié-collé un peu rapide...